Sources/SpeziAccountPhoneNumbers/DataEntry/OTCEntryView.swift

```diff
@@ -0,0 +1,137 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziAccount
+import SpeziViews
+import SwiftUI
+
+
+enum FocusPin: Hashable {
+    case pin(Int)
+}
+
+struct OTCEntryView: View {
+    @State private var viewState = ViewState.idle
+    @FocusState private var focusState: FocusPin?
+    @Environment(Account.self)
+    private var account
+    @Environment(PhoneVerificationProvider.self)
+    private var phoneVerificationProvider
+    @Environment(PhoneNumberViewModel.self)
+    private var phoneNumberViewModel
+    private let codeLength: Int
+    @State private var pins: [String]
+    @State private var resendTimeOut = 30
+    
+    private let timer = Timer.publish(every: 1, on: .main, in: .common).autoconnect()
+    
+    
+    var body: some View {
+        VStack {
+            HStack(spacing: 15) {
+                ForEach(0..<codeLength, id: \.self) { index in
+                    individualPin(index: index)
```

Comment:
Not sure how this currently behaves with VoiceOver and accessibility controls. I suspect it its quite clumsy to work with?
Would It make sense to use a `accessibilityRepresentation` modifier and replace this with a simple text field for voice over? Of course we want to apply the same modifiers like `keyboardType` there as well.